packages/protocol-perf/src/index.ts

```diff
@@ -147,23 +189,48 @@ class DefaultPerfService implements Startable, PerfService {
 
     const stream = await connection.newStream([this.protocol], options)
 
+    let lastAmountOfBytesSent = BigInt(0)
+    let lastReportedTime = Date.now()
+    let totalBytesSent = BigInt(0)
+
+    const initialStartTime = Date.now()
```

Comment:
```suggestion
    const initialStartTime = Date.now()
    
    let lastReportedTime = initialStartTime
    let totalBytesSent = BigInt(0)
```

initial and lastReportedTime should be the same for first response to ensure the most accuracy

Reply:
I get that this doesn't fix a bug or issue or anything but will set up future changes to not bite us if we do end up doing other things